tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/Tablespace.java

```diff
@@ -49,18 +51,24 @@
  */
 public abstract class Tablespace {
 
-  public static final PathFilter hiddenFileFilter = new PathFilter() {
-    public boolean accept(Path p) {
-      String name = p.getName();
-      return !name.startsWith("_") && !name.startsWith(".");
-    }
-  };
+  protected final String name;
+  protected final URI uri;
+  /** this space is visible or not. */
```

Comment:
TablespaceManager automatically adds temporary tablespaces like ``file:///``. These temporary tables are only used internally in Tajo, and they must not not visible. The visible member variable is for that.